app/components/Dashboard.jsx

```diff
@@ -35,8 +36,9 @@ class Dashboard extends Component {
     // Socket event to handle updates to consumer metrics
     socket.on('consumer', (data) => this.props.addConsumerData(data));
   }
-
+  // Render
   render() {
+    // Updated properties from store
```

Comment:
you shouldn't say "updated" here. they aren't always the "updated properties". when we first start the application, these components are still rendered. at that point in time the data hasn't been updated. I'd just leave a comment like "Destructure metric properties from our state"